contracts/protocols/mainnet/morpho/aave_v3/helpers.sol

```diff
@@ -0,0 +1,320 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+import "./interfaces.sol";
+import { DSMath } from "../../../../utils/dsmath.sol";
+
+contract MorphoHelpers is DSMath {
+    /**
+     *@dev Returns ethereum address
+     */
+    function getEthAddr() internal pure returns (address) {
+        return 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+    }
+
+    function getAWethAddr() internal pure returns (address) {
+        return 0x030bA81f1c18d280636F32af80b9AAd02Cf0854e;
+    }
+
+    function getChainlinkEthFeed() internal pure returns (address) {
+        return 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419; // should be updated after deployment
+    }
+
+    function getAaveProtocolDataProvider() internal pure returns (address) {
+        return 0x057835Ad21a177dbdd3090bB1CAE03EaCF78Fc6d; // should be updated after deployment
```

Comment:
We need Aave v3 addresses, not Aave v2.
Update from here ->
https://docs.aave.com/developers/deployed-contracts/v3-mainnet/ethereum-mainnet